components/shared/gallery/BreadcrumbsFilter.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <b-field grouped group-multiline class="filters-tag">
+    <template v-for="(value, key) in breads">
+      <div v-if="value === 'true'" :key="key" class="control">
+        <!-- NeoTag -->
+        <b-tag
+          attached
+          closable
+          aria-close-label="clear filter"
+          @close="closeTag(String(key))">
+          {{ queryMap[key] }}
+        </b-tag>
+      </div>
+    </template>
+  </b-field>
+</template>
+
+<script lang="ts" setup>
+const route = useRoute()
+const router = useRouter()
+const { $consola } = useNuxtApp()
+
+const queryMap = {
+  listed: 'Buy Now',
+  owned: 'Own',
+}
+
+const breads = computed(() => route.query)
+
+const closeTag = (key: string) => {
+  router
+    .replace({
+      path: String(route.path),
+      query: {
+        page: '1',
+        ...route.query,
```

Comment:
The `route.query` would  override `page: 1`. Is it expected?

Reply:
hmm.. it's not possible to `closeTag` when you're on page 2 since Tags are placed on top of the page and it will always be page 1? Am I right? or maybe I miss something

Reply:
I mean we should put the `page 1` after `route.query` so that it could always set the page to the first page.
```
query: {
        ...route.query,
        ...{ [key]: false },
        page: '1',
```